Client/Frontend/Browser/SessionData.swift

```diff
@@ -6,6 +6,33 @@ import Foundation
 
 import Shared
 
+private func migrate(urls: [URL]) -> [URL] {
+    return urls.compactMap { url in
+        var url = url
+        let port = AppInfo.webserverPort
+        [("http://localhost:\(port)/errors/error.html?url=", "\(InternalURL.baseUrl)/\(SessionRestoreHandler.path)?url=")
+            // TODO: handle reader pages ("http://localhost:6571/reader-mode/page?url=", "\(InternalScheme.url)/\(ReaderModeHandler.path)?url=")
```

Comment:
Is this something that we'll want to fix for v29? Should we file a follow up?

Reply:
This is more of an enhancement for our test suit. we can leave it for a later release.